webpack/assets/javascripts/react_app/components/HostDetails/Properties/index.js

```diff
@@ -121,7 +125,9 @@ const Properties = ({ hostData }) => (
                 <span id="simple-item2">{__('Organization')}</span>
               </DataListCell>,
               <DataListCell isFilled alignRight key="org-content">
-                {hostData.organization_name || <Skeleton />}
+                {hostData.organization_name || (
+                  <EmptyLine isLoading={isLoading} />
```

Comment:
Instead of the `||` operator, would it be better to handle the value rendering inside the component?

```js
 <EmptyLine isLoading={isLoading} >
  { hostData.organization_name }
</EmptyLine>
```

Reply:
I don't think it should be the loading component "responsibility" to render the data.
Also all the loading components we have already use this logic of `data? ShowData: showLoading`

Reply:
I do find it simpler for the consumer and more consistent to our stack. we already use this approach in `Loader` [component](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/Loader/index.js#L14) and with [LoadingState](https://github.com/Katello/katello/blob/master/webpack/components/LoadingState/LoadingState.js#L7) from katello.